src/test/java/core/basesyntax/service/impl/ReaderImplTest.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.service.Reader;
+import java.util.List;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class ReaderImplTest {
+    private static String EMPTY_FILE_PATH;
+    private static String NOT_EMPTY_FILE_PATH;
+    private static String NOT_EXIST_FILE_PATH;
```

Comment:
make it constants

Reply:
> make it constants

I just changed it to @BeforeAll on the advice of the mentor above